apps/extension/src/service-worker.ts

```diff
@@ -55,10 +62,12 @@ const handler = connectChannelAdapter({
   createRequestContext: req => {
     const contextValues = req.contextValues ?? createContextValues();
 
-    // dynamically initialize custodyClient, or reuse if it's already available
+    // dynamically initialize clients, or reuse if already available
     custodyClient ??= createDirectClient(CustodyService, handler, transportOptions);
+    stakingClient ??= createDirectClient(StakingService, handler, transportOptions);
```

Comment:
Adding `stakingClient` to context so that the `ViewService`'s `delegationsByAddressIndex` method can call out to the staking client (getting it via the context object)